packages/core/src/org.ts

```diff
@@ -119,10 +165,29 @@ export class OrgSecrets extends Serializable {
 export interface OrgInfo {
     id: OrgID;
     name: string;
-    owner: AccountID;
+    owner?: {
+        email: string;
+        accountId?: AccountID;
+    };
     revision: string;
 }
 
+export class ScimSettings extends Serializable {
+    @AsBytes()
+    secret!: Uint8Array;
+
+    url: string = "";
```

Comment:
@MaKleSoft this is the thing that wouldn't be saved when it was set as `url!: string;`, which I don't quite understand.

Reply:
Explained in my other comment ;)